lib/Fmt_ast.ml

```diff
@@ -853,7 +853,7 @@ and fmt_core_type c ?(box = true) ?(in_type_declaration = false) ?pro
         @@ hvbox 4
              ( hvbox 2 fmt_field
              $ fmt_docstring_padded c doc
-             $ fmt_attributes c ~pre:(str " ") ~key:"@" atrs )
+             $ fmt_attributes c ~pre:(fmt "@ ") ~key:"@" atrs )
```

Comment:
I guess we can have the `pre` depending on the `ocp-indent-compat` option so we keep the previous behavior when this option is set and JaneStreet will not impacted by the `ocp-indent` bugs